bin/cluster/management_update

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+Script to call when there is a change in the management state
+
+=head1 SYNOPSIS
+
+bin/cluster/management_update <change_type> <vrrp_ip> <state> <priority>
+
+=head1 DESCRIPTION
+
+Made for keepalived so the pf services can be adjusted depending of the running node
+
+=cut
+
+use constant INSTALL_DIR => '/usr/local/pf';
+use lib INSTALL_DIR . "/lib";
+
+use pf::log;
+
+my $logger = get_logger;
+
+$logger->info("Refreshing services for management mode");
+
+my $vrrp_ip = $ARGV[1];
+my $state = $ARGV[2];
+my $priority = $ARGV[3];
+
+if($state eq "MASTER"){
+    $logger->info("Transition to master state, starting management services.");
+    `/usr/local/pf/bin/pfcmd service pfdhcplistener start`;
```

Comment:
I agree it's more multi-master to do it this way but it adds complexity to the code especially in pfmon. It required to add the pfmon timers to the configuration overlay specific to each server and would call the expiration on them to sync on each cluster change. This makes it longer I agree but it gives a general way of handling all the services (through pf::services::manager::isEnabled) instead of adapting each service to temporarly stop while it's not master. Answer to your other question in the next comment